src/tron/tronPlugin.ts

```diff
@@ -20,6 +20,12 @@ import { encodeUriCommon, parseUriCommon } from '../common/uriHelpers'
 import { getDenomInfo } from '../common/utils'
 import { TronNetworkInfo } from './tronTypes'
 
+interface TronKeys {
+  tronMnemonic?: string
+  tronKey: string
+  derivationPath?: string
+}
```

Comment:
This type could live in tronTypes.ts. If you make a matching cleaner for it, you would use that cleaner in both `derivePublicKey` and in the currency engine, where you currently treat the keys as `any`.